content/3. Procedurele informatie/4. Techniek/Workshop Git/1. Introductie Git.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 1. Introductie Git
+taxonomie: None
+tags: 
+draft: true
+---
+## Voorwaarden deelname
+Een voorwaarde om deel te nemen aan deze workshop is het hebben van een GitHub account. Indien je dit nog niet hebt gedaan, volg dan de stappen op de [GitHub website](https://github.com)
```

Comment:
Hier moeten we even een duidelijke structuur voor afspreken. Denk dat iets als een list van voorwaarden duidelijker is dan twee zinnen. Zit je ook niet met zinsopbouw dat anders kan zijn